samples/ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -137,24 +146,35 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
         {
             await ResetEffectsAsync();
 
+            blurAmountSlider.Visibility = Visibility.Collapsed;
+            effectPropertySet = new PropertySet();
             string typeName = null;
 
             switch (effect)
             {
-                case displacementEffect: typeName = typeof(DisplacementEffect).FullName; break;
-                case rotatingTilesEffect: typeName = typeof(RotatedTilesEffect).FullName; break;
+                case displacementEffect:
+                    typeName = typeof(DisplacementEffect).FullName;
+                    break;
+                case rotatingTilesEffect:
+                    typeName = typeof(RotatedTilesEffect).FullName;
+                    break;
+                case gaussianBlurEffect:
+                    typeName = typeof(DynamicBlurVideoEffect).FullName;
+                    effectPropertySet["BlurAmount"] = blurAmountSlider.Value;
+                    blurAmountSlider.Visibility = Visibility.Visible;
+                    break;
             }
 
             if (typeName == null)
                 return;
 
-            await mediaCapture.AddVideoEffectAsync(
-                new VideoEffectDefinition(typeName, new PropertySet()),
+            await mediaCapture.AddVideoEffectAsync(new VideoEffectDefinition(typeName, effectPropertySet),
```

Comment:
Use the property bag defined at the beginning of the Task instead of an inline-defined new PropertySet . In the case of the BlurEffect selection the property bag will have a "BlurAmount" property, otherwise it is the same as before.